.circleci/config.yml

```diff
@@ -39,21 +39,15 @@ jobs:
   docs:
     <<: *common
     docker:
-      - image: cimg/python:3.6
+      - image: cimg/python:3.10
```

Comment:
I think I still like verifying that everything is verifying in the oldest supported version, rather than the newest.

It also is a simpler standard to follow, rather than "the most recent version of python, as of the last template upgrade" in each project, it's always run in the oldest version of python that any given project supports.

So this, the `docs/conf.py` and line 48 would be:
```
      - image: cimg/python:3.7
```

Reply:
All right, that makes sense. I've been running into issues lately with dev dependencies and the latest versions of python (specifically mypy and python 3.10) and they don't get caught because we're not installing dev dependencies across python versions, but maybe the better thing is to just have a new circleci command for the dev dependency installs like we do for the wheels.

Reply:
Ah, interesting, 👍🏻